irohad/main/config/config.hpp

```diff
@@ -197,10 +198,11 @@ namespace iroha {
     }
 
     inline keypair_t Config::Cryptography::keypair() const noexcept {
-      return keypair_t{iroha::pubkey_t::from_string(public_key),
-                       iroha::privkey_t::from_string(private_key)};
+      return keypair_t{
+          hexstringToArray<32>(public_key).value(),
+          hexstringToArray<64>(private_key).value()};
```

Comment:
maybe it will be better to fix `from_string` method?

Reply:
from_string parses `bytes` and returns key. `hexstringToArray` is temporary solution here, it parses `hexstring`